app/assets/javascripts/components/sidebar_group.es6.jsx

```diff
@@ -0,0 +1,28 @@
+class SidebarGroup extends React.Component {
+
+	get styles() {
+		return {
+			container: {
+				position: 'relative',
+        borderBottom: 'solid #D6D6D6 1px',
+			},
+		};
+	}
+
+  render() {
+    return(
+    	<div style={this.styles.container}>
+	    	<SidebarItem 
+	        label={'Student'} 
+	        icon={'fa fa-pencil fa-lg'}/>
+	      <SidebarItem 
+	        label={'School'}
+	        icon={'fa fa-building-o fa-lg'}/>
+	      <SidebarItem 
+	        label={'Staff'}
+	        icon={'fa fa-male fa-lg'}/>
```

Comment:
Always add a space before the `/>` when component definitions are in this form.
So, `icon={'fa fa-male fa-lg'} />`.